packages/iov-cosmos/tsconfig.json

```diff
@@ -4,9 +4,8 @@
     "baseUrl": ".",
     "outDir": "build",
     "declarationDir": "build/types",
-    "rootDir": "src"
+    "rootDir": "src",
+    "lib": ["es6", "dom"]
```

Comment:
"lib" "dom" means that we're telling the TS compiler the dom API is available in the target JS engine. This is only the case for browsers. Do we really need this option?

Reply:
I added this because amino-js seems to require it. Maybe we can define some global instead?
```
../../node_modules/@tendermint/amino-js/src/util.ts:21:49 - error TS2304: Cannot find name 'window'.

21     ({ atob, btoa, TextDecoder, TextEncoder } = window);
```

Reply:
That's a bug in the library. Or the library is browser-only.

Anyways, why are we even compiling an external .ts file? We should only consume .js and .d.ts from dependencies.

Reply:
Issue: https://github.com/cosmos/amino-js/issues/4